src/Plutarch/Extra/IsData.hs

```diff
@@ -72,34 +61,46 @@ import PlutusTx (
     fromData,
     toData,
  )
-import Prelude (
-    Bounded,
-    Enum,
-    Integer,
-    Maybe (Just, Nothing),
-    enumFrom,
-    error,
-    fmap,
-    foldr,
-    fromEnum,
-    fromInteger,
-    maxBound,
-    minBound,
-    pure,
-    toEnum,
-    toInteger,
-    ($),
-    (.),
-    (<$>),
- )
 
 --------------------------------------------------------------------------------
 -- ProductIsData
 
-{- |
-  Wrapper for deriving 'ToData', 'FromData' using the 'List' constructor of Data to represent a Product type.
-
-  Uses 'gProductToBuiltinData', 'gproductFromBuiltinData'.
+{- | Wrapper for deriving 'ToData', 'FromData' using the 'List'
+     constructor of Data to represent a Product type.
+
+     It is recommended to use `PlutusTypeDataList` when deriving
+     `PlutusType` as it provides some basic safety by ensuring
+     Plutarch types have an `Inner` type of `PDataRecord`.
```

Comment:
These should use `'`-wrapping so the Haddocks link. If they're not linkable, `@`-wrapping is what you want for monospacing. Unfortunately, Haddock syntax is a bit wonky.

Reply:
what does ``` do?

Reply:
I actually don't think it does anything?

Reply:
:O